aiida_quantumespresso/calculations/pp.py

```diff
@@ -1,27 +1,240 @@
+        # copy remote output dir, if specified
+        parent_calc_folder = self.inputs.get('parent_folder', None)
+        if parent_calc_folder is not None:
+            if isinstance(parent_calc_folder, orm.RemoteData):
+                parent_calc_out_subfolder = settings.pop('PARENT_CALC_OUT_SUBFOLDER', self._INPUT_SUBFOLDER)
+                remote_copy_list.append((
+                    parent_calc_folder.computer.uuid,
+                    os.path.join(parent_calc_folder.get_remote_path(), parent_calc_out_subfolder),
+                    self._OUTPUT_SUBFOLDER
+                ))
+            elif isinstance(parent_calc_folder, orm.FolderData):
+                for filename in parent_calc_folder.list_object_names():
+                    local_copy_list.append((
+                        parent_calc_folder.uuid,
+                        filename,
+                        os.path.join(self._OUTPUT_SUBFOLDER, filename)
+                    ))
+            elif isinstance(parent_calc_folder, orm.SinglefileData):
+                single_file = parent_calc_folder
+                local_copy_list.append(
+                    (single_file.uuid, single_file.filename, single_file.filename)
+                )
+
+        codeinfo = datastructures.CodeInfo()
+        codeinfo.cmdline_params = settings.pop('CMDLINE', [])
+        codeinfo.stdin_name = self.inputs.metadata.options.input_filename
+        codeinfo.stdout_name = self.inputs.metadata.options.output_filename
+        codeinfo.code_uuid = self.inputs.code.uuid
+
+        calcinfo = datastructures.CalcInfo()
+        calcinfo.uuid = str(self.uuid)
+        calcinfo.codes_info = [codeinfo]
+        calcinfo.local_copy_list = local_copy_list
+        calcinfo.remote_copy_list = remote_copy_list
+
+        # Retrieve by default the output file and the xml file
+        calcinfo.retrieve_list = []
+        calcinfo.retrieve_list.append(self.inputs.metadata.options.output_filename)
+        settings_retrieve_list = settings.pop('ADDITIONAL_RETRIEVE_LIST', [])
+        calcinfo.retrieve_list += settings_retrieve_list
+        calcinfo.retrieve_list += self._internal_retrieve_list
+
+        calcinfo.retrieve_singlefile_list = self._retrieve_singlefile_list
+
+        # We might still have parser options in the settings dictionary: pop them
+        # We need an instance of the parser class to get the parser options key (typically 'parser_options')
+        parser_name = self.inputs.get('metadata.options.parser_name', None)
+        if parser_name is not None:
+            Parserclass = ParserFactory(parser_name)
+            parser = Parserclass(self)
+            try:
+                parser_opts = parser.get_parser_settings_key().upper()
+                settings.pop(parser_opts)
+            except (KeyError, AttributeError):
+                # the key parser_opts isn't inside the dictionary,
+                # or the parser doesn't have a method get_parser_settings_key().
+                pass
```

Comment:
I don't think your parser supports any options,  so best to remove this

Reply:
Done